egs/tts/valle_v2/README.md

```diff
@@ -0,0 +1,169 @@
+# VALLE
+## Introduction
+This is an unofficial PyTorch implementation of VALL-E, a zero-shot voice cloning model via neural codec language modeling ([paper link](https://arxiv.org/abs/2301.02111)). 
+If trained properly, this model could match the performance specified in the original paper.
+
+## Change notes
+This is a refined version compared to the first version of VALLE in Amphion, we have changed the underlying implementation to Llama
+to provide better model performance, faster training speed, and more readable codes.
+This can be a great tool if you want to learn speech language models and its implementation.
+
+## Installation requirement 
+
+Set up your environemnt as in Amphion README (you'll need a conda environment, and we recommend using Linux). A GPU is recommended if you want to train this model yourself.
+For inferencing our pretrained models, you could generate samples even without a GPU.
+To ensure your transformers library can run the code, we recommend additionally running:
+```bash
+pip install -U transformers==4.41.2
+```
+
+<!-- espeak-ng is required to run G2p. To install it, you could refer to: 
+https://github.com/espeak-ng/espeak-ng/blob/master/docs/guide.md
+
+For Linux, it should be `sudo apt-get install espeak-ng`.
+For Windows, refer to the above link.
+If you do not have sudo privilege, you could build the library by following the last section of this readme. -->
+
+## Inferencing pretrained VALLE models
+### Download pretrained weights
+You need to download our pretrained weights from huggingface. 
+
+Script to download AR and NAR model checkpoint: 
+```bash
+huggingface-cli download amphion/valle valle_ar_mls_196000.bin valle_nar_mls_164000.bin --local-dir ckpts
+```
+Script to download codec model (SpeechTokenizer) checkpoint:
+```bash
+huggingface-cli download amphion/valle speechtokenizer_hubert_avg/SpeechTokenizer.pt speechtokenizer_hubert_avg/config.json --local-dir ckpts
+```
+
+### Inference in IPython notebook
+
+We provide our pretrained VALLE model that is trained on 45k hours MLS dataset.
+The "demo.ipynb" file provides a working example of inferencing our pretrained VALL-E model. Give it a try!
```

Comment:
Where is the "demo.ipynb"?

Reply:
uploaded. It was ignored by Amphion by its gitignore.